packages/quick_actions/quick_actions_android/example/android/app/src/androidTest/java/io/flutter/plugins/quickactionsexample/QuickActionsTest.java

```diff
@@ -20,4 +54,108 @@ public void imagePickerPluginIsAdded() {
           assertTrue(activity.engine.getPlugins().has(QuickActionsPlugin.class));
         });
   }
+
+  @Test
+  public void appShortcutsAreCreated() {
+    // Arrange
+    List<Shortcut> expectedShortcuts = createMockShortcuts();
+
+    // Act
+    ShortcutManager shortcutManager =
+        (ShortcutManager) context.getSystemService(Context.SHORTCUT_SERVICE);
+    List<ShortcutInfo> dynamicShortcuts = shortcutManager.getDynamicShortcuts();
+    Object[] shortcuts = dynamicShortcuts.stream().map(Shortcut::new).toArray();
```

Comment:
I may be missing some context here, but can you just use the `ShortcutInfo`s to test versus creating this new `Shortcut` object?

Reply:
I meant to keep the test variants created from `createMockShortcuts` as simple as possible, without the need to go through the complexity of creating a `ShortcutInfo`.

Reply:
What is that complexity? If doable, I would suggest testing with `ShortcutInfo`s for clarity since this is the object actually used in the plugin.

Reply:
I have updated the code.